### PR TITLE
Issue #35

### PR DESCRIPTION
**Pull request by hudsongeovane**
If the current file has been changed, the file name is changed to "file.tex*".
Besides, when the user clicks on "SAVE", now the file is automatically overwritten (as in most editors). The first "SAVE" click will NOT overwrite the "welcome.tex" file, the program will ask the user a new filename (and path), and then, the next clicks will overwrite that file.
Obviously a new "SAVE AS" button has to be created, in order to make possible to the user save the content in other file.

### CODE REVIEW

File: OpenLaTeXStudio/EditorModule/src/latexstudio/editor/menu/SaveFile.java [hunk 1]
```diff
@@ -43,7 +43,8 @@
     @Override
     public void actionPerformed(ActionEvent e) {                
         String content = etc.getEditorContent();
-        File file = FileChooserService.getSelectedFile("tex", "TeX files", DialogType.SAVE, true);
+        File file = etc.getCurrentFile();
+        if (file == null) file = FileChooserService.getSelectedFile("tex", "TeX files", DialogType.SAVE, true);
```
Comment (sebbrudzinski) [inline OpenLaTeXStudio/EditorModule/src/latexstudio/editor/menu/SaveFile.java]: We should not alter this behaviour until "Save as" option becomes available. This change blocks the ability for the user to save as a new file. We already have a ticket for "save as" option, so this should be done there.

**Comment by hudsongeovane** [comment]
Right! So I think it is better to fix this when Save As (and Save) are avaiable.

Thanks for reviewing!

**Comment by hudsongeovane** [comment]
Well, I have implemented the "Save As" button. I just need a new icon. While I don't have it, the same icon of Save File is used.

**Comment by sebbrudzinski** [comment]
@hudsongeovane - we cannot have two identical icons displayed in the toolbar. For now, you can just delete the icon for save as, and we can add one at a later point of time.

**Comment by hudsongeovane** [comment]
Well, I added an icon. It can be changed afterwards.

**Pull request merged**